model/aws/registry.yaml

```
@@ -8,7 +8,7 @@ groups:
       - id: aws.request_id
         type: string
         stability: experimental
-        brief: "The AWS request ID as returned in the response headers `x-amz-request-id` or `x-amz-requestid`."
+        brief: "The AWS request ID as returned in the response headers `x-amz-request-id` or `x-amzn-requestid`."
```

it seems both forms are supported

https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/API_dataplane_GetMedia.html

> x-amz-RequestId HTTP header – if you want to report an issue to AWS, the support team can better diagnose the problem if given the Request Id.

So maybe 
```suggestion
        brief: "The AWS request ID as returned in the response headers `x-amz-request-id`, `x-amz-requestid`, or `x-amzn-requestid`."
```

?

---

Seems so yes, AWS SDK source code confirms. Updated and corrected commit message + PR description.